assets/src/edit-story/components/panels/test/layerStyle.js

```diff
@@ -60,4 +60,24 @@ describe('Panels/LayerStyle', () => {
     fireEvent.change(input, { target: { value: '23' } });
     expect(pushUpdate).toHaveBeenCalledWith({ opacity: 23 });
   });
+
+  it('should set min to opacity value on change', () => {
+    const { getByRole, submit } = renderLayerStyle([{ id: 1, opacity: 49 }]);
+    const input = getByRole('textbox', { name: 'Opacity in percentage' });
+    fireEvent.change(input, { target: { value: null } });
+    const submits = submit({ opacity: null });
+    expect(submits[1]).toStrictEqual({
```

Comment:
`submits[1]` doesn't really mean anything to me here. Can we name this better? Or at least add a comment about what this refers to?